decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -36,6 +36,9 @@ extra_admin_link(
 
 <% content_for :aside do %>
   <%= render partial: "proposal_aside" %>
+  <div class="suggested-content mt-10 hidden md:block">
+    <%= cell "decidim/proposals/content_suggestion_section", @proposal %>
+  </div>
```

Comment:
You could sorounding block dirrectly in the content_suggestion cell ( so that we can control rendering ). Also i do not think you need the `suggested-content` class

```suggestion
    <%= cell "decidim/proposals/content_suggestion_section", @proposal, container_classes: " hidden md:block" %>
```